Utilities/CodeGeneration/EdFi.Ods.CodeGen/Providers/Impl/AssemblyDataProvider.cs

```diff
@@ -70,7 +72,26 @@ public IEnumerable<AssemblyData> Get()
                 .Select(Create)
                 .ToList();
 
-            if (_includePluginsProvider.IncludePlugins())
+            var extensionLocationPaths = _extensionLocationPluginsProviderProvider.GetExtensionLocationPlugins();
+
+            if (extensionLocationPaths != null && extensionLocationPaths.Any())
```

Comment:
This if statement is redundant since you are doing a foreach loop

Reply:
Fixed it